persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/impl/table/AbstractTableManager.java

```diff
@@ -72,7 +72,7 @@
       this.selectIdRowSql = initSelectIdRowSql();
       this.deleteRowSql = initDeleteRowSql();
       this.loadAllRowsSql = initLoadAllRowsSql();
-      this.countRowsSql = initCountNonExpiredRowsSql();
+      this.countNonExpiredRowsSql = initCountNonExpiredRowsSql();
```

Comment:
As this will be the only change to the production files, I think we should revert it.